yarn.lock

```diff
@@ -3,21 +3,21 @@
 
```

Comment:
Why do we have a `yarn.lock` and a `package-lock.json`? We should just use one of the two, but not both. https://stackoverflow.com/a/46590427

Reply:
If i remove the file solves the problem? Or do i need to do something else?

Reply:
Yes, you can remove one of the two